src/main/java/ratpack/zipkin/internal/ZipkinHttpClientImpl.java

```diff
@@ -135,11 +135,11 @@ private void resultWithSpan(Result<?> result, StatusCode statusCode, Span span)
             int status = statusCode.getStatusCode();
             if (status < 200 || status > 399) {
                 span.tag(TraceKeys.HTTP_STATUS_CODE, String.valueOf(status));
-            } else if (status > 499) {
```

Comment:
I think this might have been a minor bug - I'm assuming that we want the `Constants.ERROR` tag *in addition* to the `HTTP_STATUS_CODE` tag.